spec/controllers/games_controller_spec.rb

```diff
@@ -136,4 +136,186 @@
       expect(response).to redirect_to(games_url)
     end
   end
+
+  #Adds white pieces to board
+
+  describe "populate board" do
+    it 'Adds correct number of white Pawns to the board' do
+     game = Game.create!
+     game.populate_board!
+
+     expect(game.pieces.where(type: 'Pawn', color: 'White')).to eq (game.pieces.where(position_x: 6, position_y: 0))
+     expect(game.pieces.where(type: 'Pawn', color: 'White')).to eq (game.pieces.where(position_x: 6, position_y: 1))
+     expect(game.pieces.where(type: 'Pawn', color: 'White')).to eq (game.pieces.where(position_x: 6, position_y: 2))
+     expect(game.pieces.where(type: 'Pawn', color: 'White')).to eq (game.pieces.where(position_x: 6, position_y: 3))
+     expect(game.pieces.where(type: 'Pawn', color: 'White')).to eq (game.pieces.where(position_x: 6, position_y: 4))
+     expect(game.pieces.where(type: 'Pawn', color: 'White')).to eq (game.pieces.where(position_x: 6, position_y: 5))
+     expect(game.pieces.where(type: 'Pawn', color: 'White')).to eq (game.pieces.where(position_x: 6, position_y: 6))
+     expect(game.pieces.where(type: 'Pawn', color: 'White')).to eq (game.pieces.where(position_x: 6, position_y: 7))
```

Comment:
Does this pass?

Reply:
@mscoutermarsh according to rspec, everything passes, but I'm not exactly sure how, since there was a lot of work that needed to be done to get a piece to actually create in the DB on the render-pieces branch. I just created a new game on this branch, and I can see in Terminal that the pieces didn't create. I think I'd like to get the other branch merged, and then rebase this branch before trying to fix it here. What do you think?